### PR TITLE
Implement #95

### LINKED ISSUE #95: Ask for confirm for /game stop

### PR DESCRIPTION
**Pull request by Restioson**
Closes #95

### CODE REVIEW

File: src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java [hunk 1]
```diff
@@ -77,6 +77,9 @@ public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
                 .then(literal("stop")
                     .requires(source -> source.hasPermissionLevel(2))
                     .executes(GameCommand::stopGame)
+                        .then(literal("confirm")
+                        .requires(source -> source.hasPermissionLevel(2))
```
Comment (Gegy) [inline src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java:81]: I believe the parenthesis is wrong here- the `executes(stopGameConfirmed)` will be attached to the root stop node

Comment (Restioson) [reply src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java:81]: It works in game and you can reformat it to 
```java
then(
    literal("stop")
        .requires(source -> source.hasPermissionLevel(2))
        .executes(GameCommand::stopGame)
                .then(
                    literal("confirm")
                    .requires(source -> source.hasPermissionLevel(2))
                        .executes(GameCommand::stopGameConfirmed)
                )
)
```

Comment (Gegy) [reply src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java:81]: Oops, ignore me

File: src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java [hunk 2]
```diff
@@ -292,6 +295,27 @@ private static int stopGame(CommandContext<ServerCommandSource> context) throws
 
         PlayerSet playerSet = gameSpace.getPlayers().copy();
```
Comment (Gegy) [inline src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java:296]: This doesn't need to copy anymore since it doesn't do anything besides check size

**Pull request merged**